apps/condo/domains/ticket/utils/serverSchema/TicketDocumentGeneration/completionWorks.js

```diff
@@ -0,0 +1,145 @@
+const { getAddressDetails } = require('@condo/domains/property/utils/serverSchema/helpers')
+const { TICKET_DOCUMENT_GENERATION_TASK_FORMAT } = require('@condo/domains/ticket/constants/ticketDocument')
+
+
+const DATE_FORMAT = 'DD.MM.YYYY'
+const formatDate = (date, timeZone, format = DATE_FORMAT) => {
+    return dayjs(date).tz(timeZone).format(format)
+}
+
+const buildExportWordFile = async ({ task, documentData, locale, timeZone }) => {
+    const { id, ticket } = task
+
+    const { stream } = await buildExportExcelFile({
+        templatePath: `./domains/ticket/templates/ticketDocumentGenerationTemplates/completionWorks/${locale}.docx`,
+        replaces: documentData,
+    })
+
+    return {
+        stream,
+        filename: `completion_works_ticket_${ticket.id}_${formatDate(undefined, timeZone, 'DD_MM_YYYY')}.docx`,
+        mimetype: DOCX_FILE_META.mimetype,
+        encoding: DOCX_FILE_META.encoding,
+        meta: {
+            listkey: 'TicketDocumentGenerationTask',
+            id,
+        },
+    }
+}
+
+const renderMoney = (amount, currencyCode, locale) => {
+    const options = { style: 'currency', currency: currencyCode }
+    const numberFormat = new Intl.NumberFormat(locale, options)
+    const parts = numberFormat.formatToParts(amount)
+    return parts.map((part) => part.value)
+        .slice(0, -2) // remove currency symbol
+        .join('')
+}
+
+const generateTicketDocumentOfCompletionWorks = async ({ task, baseAttrs, context, locale, ticket, organization }) => {
+    const { format, timeZone: timeZoneFromUser } = task
+
+    const timeZone = normalizeTimeZone(timeZoneFromUser) || DEFAULT_ORGANIZATION_TIMEZONE
+    const printDate = dayjs().tz(timeZone)
+
+    const property = await getById('Property', ticket.property)
+    const { renderData, streetPart, settlement, housePart, cityType, cityName } = getAddressDetails(get(property, 'addressMeta', ticket.propertyAddressMeta))
+
+    const contact = ticket.contact
+        ? await getById('Contact', ticket.contact)
+        : null
+
+    const employee = organization.id && ticket.executor
+        ? await getByCondition('OrganizationEmployee', {
+            organization: { id: organization.id },
+            user: { id: ticket.executor },
+        })
+        : null
+
+    const invoices = await Invoice.getAll(context, {
+        ticket: { id: ticket.id },
+        deletedAt: null,
+    }, {
+        sortBy: ['createdAt_ASC'],
+    })
+
+    const currencyCode = get(invoices, '0.currencyCode') || DEFAULT_INVOICE_CURRENCY_CODE
+
+    const listOfWorks = invoices.reduce((acc, invoice) => {
+        const rows = Array.isArray(invoice.rows) ? invoice.rows : []
+        acc.push(...rows.map(row => {
+            const price = parseFloat(row.toPay)
+
+            return {
+                name: row.name || '',
+                count: String(row.count) || '',
+                price: !Number.isNaN(price) ? renderMoney(price, currencyCode, locale) : '',
```

Comment:
If `price` === 0 and `isMin` = true it's mean that it's a `Contract` price. Probably, you need to support it here

Reply:
fixed. if `isMin` = `true` then print nothing